config/ftbquests/quests/chapters/mekanism.snbt

```diff
@@ -676,19 +674,22 @@
 			]
 		}
 		{
-			title: "Ore Tripling"
+			title: "3rd Tier Ore Processing"
 			icon: "mekanism:electrolytic_core"
 			x: 0.0d
 			y: 4.0d
 			subtitle: "Not Enough Minerals"
 			description: [
-				"With enough effort, Mekanism allows for the extraction of up to five ingots per ore, but this is a very involved process and will require serious effort to set up. "
+				"With enough effort, Mekanism allows for the extraction of up to 3 (3.33 exactly) ingots per ore, but this is a very involved process and will require serious effort to set up."
 				""
-				"However, between doubling and quintupling, it can also triple or quadruple yields from ores. Tripling is very attainable early on and well worth setting up."
+				"However, between increasing yield to 1.33 and 3.33, it can also increase it to 2 or 2.66. Doubling with fortune pick is very attainable early on and well worth setting up."
```

Comment:
Wouldn't even call out the worse rates here. If it can still do 3x and 4x, just tell them. Also not the place to mention fortune mining as that's unrelated to Mek processing.